src/pages/Admin/AdminVenuePreview.tsx

```diff
@@ -18,16 +18,24 @@ export const AdminVenuePreview: React.FC<AdminVenuePreview> = ({
         return (
           <div>
             <span className="title">iFrame URL</span>
-            <span className="content">{venue.iframeUrl}</span>
+            <span className="content">
+              <a href={venue.embedIframeUrl}>{venue.embedIframeUrl}</a>
+            </span>
           </div>
         );
-      default:
+      case VenueTemplate.zoomroom:
         return (
           <div>
             <span className="title">Zoom URL</span>
-            <span className="content">{venue.zoomUrl}</span>
+            <span className="content">
+              <a href={venue.zoomUrl} target="_blank" rel="noopener noreferrer">
+                {venue.zoomUrl}
+              </a>
+            </span>
           </div>
         );
+      default:
+        return <div></div>;
```

Comment:
That's not an amazing default. Could we have `Sorry we can't show you this experience` And a button `G back to the playa` or something?

Reply:
this is just for the last line in the preview: if it's an ArtPiece there is a link to the iFrame, if it's a ZoomRoom there is a link to the zoom url, if it's neither of them (camp) it defaults to nothing (there is no line with extra information)